nodes/List.py

```diff
@@ -1 +1,25 @@
+"""
+List astroid node
```

Comment:
We talked about trying to get a "Store" context for lists. I wonder if this is achieved by the assignment statement

```python
[x, y] = [1, 2]
```